vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExistingSamplesPreventGroupEdit.java

```diff
@@ -0,0 +1,41 @@
+package life.qbic.datamanager.views.projects.project.experiments.experiment;
+
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import life.qbic.datamanager.views.notifications.NotificationDialog;
+
+/**
+ * Notifies the user that samples exist in the experiment.
+ * <p>
+ * This dialog is to be shown when editing experimental groups is impossible as samples are
+ * present.
+ */
+public class ExistingSamplesPreventGroupEdit extends NotificationDialog {
```

Comment:
NIT: I think having the component type in the name helps understand what it is for within the structure tree 🤔 
```suggestion
public class ExistingSamplesPreventGroupEditDialog extends NotificationDialog {
```